packages/amplify-graphql-validate-transformer/src/validators.ts

```diff
@@ -57,6 +58,21 @@ const validateNoListFieldValidation = (fieldNode: FieldDefinitionNode): void =>
   }
 };
 
+/**
+ * Validates that `@validate` directive is not placed before a `@default` directive.
+ * @param fieldNode - The field definition node that the directive is applied to
+ * @param currentDirective - The current `@validate` directive node
+ */
+const validateOrderingWithDefaultDirective = (fieldNode: FieldDefinitionNode, currentDirective: DirectiveNode): void => {
+  const directives = fieldNode.directives!;
+  const validateDirectiveIndex = directives.indexOf(currentDirective);
+  const defaultDirectiveIndex = directives.findIndex((d) => d.name.value === 'default');
```

Comment:
nit, really just for future awareness that it exists.

```suggestion
  const defaultDirectiveIndex = directives.findIndex((d) => d.name.value === DefaultDirective.name);
```

If you want to make this change, don't just commit the suggestion. You'd also need to import `DefaultDirective` from the amplify-graphql-directives package.